### PR TITLE
Create Touch area component #15

### PR DESCRIPTION
**Pull request by what-the-hub**
**Refs:**

[Create PR template](https://github.com/what-the-hub/vue-game/issues/21)

**Proposed Changes:**

- Update path to template to fix suggestion in github `docs/PULL_REQUEST_TEMPLATE.md`

### CODE REVIEW

File: src/components/Board.vue [hunk 1]
```diff
@@ -21,34 +19,33 @@
...
-    Arrow
+    Arrow, Areas
   }
 })
 export default class Board extends Vue {
   isActive: boolean = false // flag for start and finish game
   safeLoop: number = 0
+  areasPositions!: IAreasPositions
+
+  get positions (): IAreasPositions {
```
Comment (silchencko) [inline src/components/Board.vue:39]: Do you need this getter? Can you just use `areasPositions`?

File: src/gameComponents/Areas.vue [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    good: '',
+    excellent: ''
+  }
+
+  mapStyles: {[index: string]: string} = {
+    good: 'good-animation',
+    excellent: 'excellent-animation'
+  }
+
+  get lastStyle () {
```
Comment (silchencko) [inline src/gameComponents/Areas.vue:28]: The same question. Do you need this getter, if it just returns data?

File: src/components/Board.vue (hunk 1 above)
Comment (what-the-hub) [reply src/components/Board.vue:39]: @silchencko I'll check it out. Thanks

File: src/gameComponents/Areas.vue (hunk 2 above)
Comment (what-the-hub) [reply src/gameComponents/Areas.vue:28]: @silchencko I'll check it too

**Review by silchencko — APPROVED** [review]

**Pull request merged**